modules/@apostrophecms/rich-text-widget/ui/apos/components/AposTiptapLink.vue

```diff
@@ -9,7 +9,13 @@
       :icon="tool.icon ? tool.icon : false"
       :modifiers="['no-border', 'no-motion']"
     />
+    <!--
+      TODO: Confirm if the `editor` object passed into this `editor-menu-bubble`
```

Comment:
I briefly tried passing in a fresh editor to this and it worked, but I didn't want to create a can of worms.